paddle/fluid/framework/op_desc.cc

```diff
@@ -373,6 +373,11 @@ std::vector<std::string> OpDesc::AttrNames() const {
   return retv;
 }
 
+void OpDesc::RemoveAttr(const std::string &name) {
+  attrs_.erase(name);
+  need_update_ = true;
+}
```

Comment:
Why do you add this API? We should not have this API, because we should not allow the user to change the Attr after this Op has been instanced.

Reply:
https://github.com/PaddlePaddle/Paddle/pull/15841/files#diff-91ee3921323472bc3550bcf1709eb34fR744

Reply:
Remove op_role_var attribute.